src/e2e/resources/test_original.properties

```diff
@@ -0,0 +1,107 @@
+#-----------------------------------------------------------------------------
+# This file contains some configuration values used during testing.
+# It should be placed in src\e2e\resources
+# Use '\' to escape ':' e.g., http\://google.com
+#-----------------------------------------------------------------------------
```

Comment:
I believe this file is not supposed to be part of your PR. Please remove it and we can go ahead with merging your changes

Reply:
@madanalogy Oops! Sorry about that. I just deleted it from the commit.